libs/unity/library/Runtime/Scripts/Media/UwpUtils.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using UnityEngine;
+
+#if UNITY_WSA && !UNITY_EDITOR
+using global::Windows.UI.Core;
+using global::Windows.Foundation;
+using global::Windows.Media.Core;
+using global::Windows.Media.Capture;
+using global::Windows.ApplicationModel.Core;
+
+namespace Microsoft.MixedReality.WebRTC.Unity
+{
+    internal static class UwpUtils
+    {
+        internal static async Task<bool> RequestAccessAsync(StreamingCaptureMode mode)
+        {
+            // Note that the UWP UI thread and the main Unity app thread are always different.
+            // https://docs.unity3d.com/Manual/windowsstore-appcallbacks.html
+            Debug.Assert(!UnityEngine.WSA.Application.RunningOnUIThread());
+            var permissionTcs = new TaskCompletionSource<bool>();
+            UnityEngine.WSA.Application.InvokeOnUIThread(() =>
+            {
+                // Request UWP access to audio capture. The OS may show some popup dialog to the
+                // user to request permission. This will succeed only if the user grants permission.
```

Comment:
```suggestion
                // Request UWP access to audio or video capture. The OS may show some popup dialog to the
                // user to request permission. This will succeed only if the user grants permission.
```

Reply:
Fixed